docs/src/config/sidebars.ts

```diff
@@ -413,6 +413,15 @@ const sidebars: SidebarsConfig = {
             }
           ]
         },
+        {
+          multitenant: [
```

Comment:
This should be placed behind the `m365group` group.

Reply:
fixed